datastore_dev.yml

```diff
@@ -0,0 +1,19 @@
+---
+- hosts: datastore-dev
+  remote_user: root
+  gather_facts: no
+  roles:
+    - common/python2
+    - common/postgres
+    - common/create_user
+    - datastore/datastore_setup
+
+  environment:
+    DATABASE_URL: "{{ db_env_var }}"
```

Comment:
Variable name could be clarified to better demonstrate what it is to be used for than just being an environment variable for a database.

Reply:
This needs some thought.